swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory 
+        self.item = Item()
+    
+    def add(self,item):
+        self.inventory.append(item)
+        return item 
+    
+    def remove(self, item):
+        if item in self.inventory: 
+            self.inventory.remove(item)
+            return item
+        else: 
+            return False 
+
+    def get_by_category(self, category):
+        items = [] 
+        for item in self.inventory: 
+            if item.category == category: 
+                items.append(item)
+        return items 
+# wave 3  
+    def swap_items(self, friend, my_item, their_item):
+
+        if my_item not in self.inventory or their_item not in friend.inventory: 
+            return False
+        #else:
+        self.inventory.remove(my_item)  
+        friend.inventory.append(my_item) 
+        friend.inventory.remove(their_item)
+        self.inventory.append(their_item)  
```

Comment:
this works fine! another thing we could do is use the instance methods `add` and `remove` we've made.

```suggestion
        self.remove(my_item)  
        friend.add(my_item) 
        friend.remove(their_item)
        self.add(their_item)  
```